.github/actions/setup/action.yml

```
@@ -5,13 +5,11 @@ inputs:
   setup-mops:
     description: Whether to setup mops
     required: false
-    default: true
-    type: boolean
+    default: "true"
```

Below you use single quotes `'` but here `"`. Why?

---

https://docs.github.com/en/actions/reference/evaluate-expressions-in-workflows-and-actions#literals
here is an example where `'` is used `${{ 'It''s open source!' }}`. `"` will throw an error

---

here I could probably use `'` too 
```suggestion
    default: 'true'
```
But I cannot omit `'`/`"`. 
If you want we can replace `"` with `'`.